include/experimental/group/gemm/impl/int4_dequantize_xe.hpp

```diff
@@ -176,12 +190,20 @@ class gemm_t<
 
   // note: plane format, row-major
   // note: 4bit x 2, row-major
-  using matB_tile_desc_t = subgroup::tile_desc_t<
-      tile_size_x_b / pack_ratio,
-      tile_size_y_b,
-      block_size_x_b / pack_ratio,
-      block_size_y_b,
-      reg_layout::tiled>;
+  using matB_tile_desc_t = std::conditional_t<
+      is_col_major_b,
+      subgroup::tile_desc_t<
+          tile_size_x_b,
+          tile_size_y_b / pack_ratio,
+          block_size_x_b,
+          block_size_y_b / pack_ratio,
+          reg_layout_b>,
+      subgroup::tile_desc_t<
+          tile_size_x_b / pack_ratio,
+          tile_size_y_b,
+          block_size_x_b / pack_ratio,
```

Comment:
will row/col major weight share the same block_size_x_b in policy?
if true, will this behavior affect simd lane utilization?

Reply:
This is a problem. But if it is divided into two policies, it will increase the template and increase the compilation time of ipex. I will give a solution to this problem as soon as possible.